frontend/src/App.js

```diff
@@ -13,6 +19,10 @@ class App extends Component {
           <div>
             <Route exact path = '/' component = {LandingPage} />
             <Route exact path = '/create' component = {CreateRecipe} />
+            <Route exact path="/recipes" component={RecipeList}/>
+            <Route exact path ="/recipes/:id" component = {IndividualRecipe}/>
```

Comment:
Make sure you keep the spacing consistent
```suggestion
            <Route exact path ="/recipes/:id" component={IndividualRecipe}/>
```